.pre-commit-config.yaml

```diff
@@ -1,21 +1,21 @@
 exclude: ^tests/data/
 repos:
-  - repo: https://gitlab.com/pycqa/flake8.git
```

Comment:
This file should not be modified in this PR.

Reply:
A mistake when I temporally debug on windows. just fixed.